.github/workflows/ci.yml

```diff
@@ -176,8 +176,8 @@ jobs:
           restore-keys: |
             Examples-v${{ env.RESET_EXAMPLES_CACHE }}-${{ steps.version.outputs.PYFLUENT_VERSION }}-${{ env.DOC_DEPLOYMENT_IMAGE_TAG }}
 
-      - name: Build Documentation
-        run: make build-doc
+      - name: Build Source Documentation
+        run: make build-doc-source
```

Comment:
Disable the release doc deploy step with this job for now as it will push incomplete documentation. We'll need a similar strategy as dev doc build to have the complete release doc (can be done in a separate PR).

Reply:
Done. Thank you.